src/Phinx/Db/Table/Column.php

```diff
@@ -626,7 +682,10 @@ protected function getValidOptions()
             'properties',
             'values',
             'collation',
-            'encoding',
+        'encoding',
+        /* James Duncan Added seed and increment for MS SQL identity columns */
+        'seed',
+        'increment',
```

Comment:
- Wrong indentation
- Comment unnecessary, there is `git blame` to get this information.

Reply:
Done